docs/docs/sourcing-from-builder-io.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Sourcing from Builder.io
+---
+
+[Builder.io](https://builder.io) has all the benefits of a modern headless CMS platform, plus the power of an easy to learn drag and drop editor that enables everyone to edit more than just content. Builder + Gatsby empowers your entire team to create performant and fully customizable pages quickly.
+
+<img src="https://imgur.com/HjBWIbv.gif" alt="Editor example" />
+
+---
+
+## Quick Start
```

Comment:
Looking at this content flow, I understand that there are two ways to get started with sourcing from BuilderIO: with the starter and following these instructions:

- Grab a free account at builder.io, this link will set you up with all the models used in the starter and some sample content to get you started.

- Find your public API key and add it to the src/config.js

However, these are not stated in this guide. Can we reduce the links to external sources by adding these instructions here or maybe in a prerequisite section?

Reply:
make sense I'll update